### ISSUE TITLE
Is this still being maintained? 

### ISSUE THREAD

**Issue by rob-w-smith** [Volnus/Gauge-Performance-Pack#1]
Would be good to know if this is still being maintained and operational to the latest Gauge version (6.1.16)?

Thanks

**Comment by Volnus** [comment]
Yes it still works for my website and if you have anything you want to add or a suggestion feel free to ask \0 but its still working on my site that is using Gauge

**Comment by rob-w-smith** [comment]
Great - downloading as we speak. Do you have the auto-optimize exclusion lists also? I think you may have forgotten to include these in your optimization video text?

**Comment by Volnus** [comment]
The latest version of Autoptimize actually should work by default. 

In the JS exclusion list you only need to make sure that jquery.js is in there and the site should function properly.

**Comment by rob-w-smith** [comment]
Okay I'm on 6.1.15...When I enable the plugin it just leaves a white page on the homepage. I've cleared caches and nothing changes?

**Comment by Volnus** [comment]
I am not getting any errors for me on a live site. Are you using the latest version of W3TC? There have been issues with it try disabling it

**Comment by w1bbsy** [comment]
Yeah, not working for me either - just blanks the screen. Im not using W3 as I use Raidboxes.eu who do caching on their platform. 

Using Autooptimize and keycdn on an SSL/TLS domain.
